extraConfigSriov.py

```diff
@@ -38,6 +40,45 @@ def _sno_make_deploy(
     client.oc("create namespace openshift-sriov-network-operator")
     client.oc("label ns --overwrite openshift-sriov-network-operator " "pod-security.kubernetes.io/enforce=privileged " "pod-security.kubernetes.io/enforce-version=v1.24 " "security.openshift.io/scc.podSecurityLabelSync=false")
 
+    ocp_version = version
+    if ocp_version is not None:
+        v, number_of_subs_made = re.subn("^(4[.][0-9]+).*$", "\\1", ocp_version)
+        if number_of_subs_made == 1:
+            ocp_version = v
+        else:
+            ocp_version = None
+
+    for ekey in (
+        "NETWORK_RESOURCES_INJECTOR_IMAGE",
+        "OVS_CNI_IMAGE",
+        "SRIOV_CNI_IMAGE",
+        "SRIOV_DEVICE_PLUGIN_IMAGE",
+        "SRIOV_INFINIBAND_CNI_IMAGE",
+        "SRIOV_NETWORK_CONFIG_DAEMON_IMAGE",
+        "SRIOV_NETWORK_OPERATOR_IMAGE",
+        "SRIOV_NETWORK_WEBHOOK_IMAGE",
+    ):
+        # The caller can set the environment variable for deployment.
+        x = os.environ.get(ekey)
```

Comment:
Very nice detail. Important and it makes this script really usable. For SRIOV development.

Although your images will unfortunately be obsoleted over time... We really need the ART team to build a public image.

Reply:
> Although your images will unfortunately be obsoleted over time

That's the main reason why this script exist. I can just re-run the script, and it pushes the latest HEAD.

Also, older branches like `release-4.14` are rather inactive...

Reply:
> I can just re-run the script

Not just me! It's (supposed to be) relatively easy, to do it yourself and push them to a local repository or to your own quay.io.